app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
+
+  def index
+  end
+
```

Comment:
prefer clear unused lines

```suggestion
  def index
  end
```